### PR TITLE
Doc: Clarify cursor position for setting debugger breakpoints

### PR DESCRIPTION
**Pull request by jfehrle**
See https://github.com/coq/coq/pull/17382#issuecomment-1471517325

@herbelin Perhaps you would review and merge?  Reviews for trivial doc improvements have been very slow lately.  Thanks.

### CODE REVIEW

File: doc/sphinx/practical-tools/coqide.rst [hunk 1]
```diff
@@ -559,7 +559,7 @@ will be stopped at the left hand edge of its range.
 The function keys are listed, for the moment, with one exception, in the `Debug` menu:
 
 Toggle breakpoint (F8)
-  Position the cursor on the first character of the tactic name in an Ltac
+  Position the cursor before the first character of the tactic name in an Ltac
```
Comment (herbelin) [inline doc/sphinx/practical-tools/coqide.rst:562]:
```suggestion
  Position the cursor just before the first character of the tactic name in an Ltac
```
???

Comment (jfehrle) [reply doc/sphinx/practical-tools/coqide.rst]: Sounds find, updated.

**Review by herbelin — APPROVED** [review]
Thanks

**Comment by herbelin** [comment]
@coqbot merge now

**Comment by herbelin** [comment]
@coqbot merge now

**Pull request merged**